das/worker.go

```diff
@@ -43,10 +43,18 @@ func (w *worker) run(
 	log.Debugw("start sampling worker", "from", w.state.From, "to", w.state.To)
 
 	for curr := w.state.From; curr <= w.state.To; curr++ {
+		startGet := time.Now()
 		// TODO: get headers in batches
 		h, err := getter.GetByHeight(ctx, curr)
 		if err == nil {
-			err = sample(ctx, h)
+			log.Debugw("got header from header store", "height", h.Height, "hash", h.Hash(),
+				"square width", len(h.DAH.RowsRoots), "data root", h.DAH.Hash(), "finished (s)", time.Since(startGet))
+
+			startSample := time.Now()
+			if err = sample(ctx, h); err == nil {
+				log.Debugw("sampled header", "height", h.Height, "hash", h.Hash(),
+					"square width", len(h.DAH.RowsRoots), "data root", h.DAH.Hash(), "finished (s)", time.Since(startSample))
```

Comment:
One last thing. Let's make it more readable by following Uber's Go guidelines and [reduce nesting](https://github.com/uber-go/guide/blob/master/style.md#reduce-nesting).

Reply:
+1